queue/src/fiskaltrust.Middleware.Queue.SQLite/fiskaltrust.Middleware.Queue.SQLite.csproj

```diff
@@ -93,4 +115,36 @@
         <Copy SourceFiles="@(SQLITE_OSX_X64)" DestinationFolder="$(TargetDir)\runtimes\osx-x64\%(RecursiveDir)" SkipUnchangedFiles="true" />
     </Target>
 
-</Project>
+    <ItemGroup>
+        <Content Include="bin\Release\net461\publish\**\*.config" Pack="true" PackagePath="lib\net461" />
+        <Content Include="bin\Release\net461\publish\**\*.exe" Pack="true" PackagePath="lib\net461" />
+        <Content Include="bin\Release\net461\publish\**\*.dll" Pack="true" PackagePath="lib\net461" />
+        <Content Include="bin\Release\net461\x64\SQLite.Interop.dll" Pack="true" PackagePath="lib\net461\x64" />
+        <Content Include="bin\Release\net461\x86\SQLite.Interop.dll" Pack="true" PackagePath="lib\net461\x86" />
+    
+        <Content Include="bin\Release\netstandard2.0\publish\**\*.config" Pack="true" PackagePath="lib\netstandard2.0" />
+        <Content Include="bin\Release\netstandard2.0\publish\**\*.exe" Pack="true" PackagePath="lib\netstandard2.0" />
+        <Content Include="bin\Release\netstandard2.0\publish\**\*.dll" Pack="true" PackagePath="lib\netstandard2.0" />
+    
+        <Content Include="bin\Release\netstandard2.1\publish\fiskaltrust.Middleware.Contracts.dll" Pack="true" PackagePath="lib\netstandard2.1" />
+        <Content Include="bin\Release\netstandard2.1\publish\fiskaltrust.Middleware.Localization.QueueDE.dll" Pack="true" PackagePath="lib\netstandard2.1" />
```

Comment:
Are we this specific here intentionally? Can we use wildcards like above, maybe?